Automattic/jetpack
# Standardize block description phrasing

Fixes https://github.com/Automattic/jetpack/issues/32603

## Proposed changes:

Updates the description of some blocks so that all blocks follow the same phrasing, e.g. start with a verb in the infinitive form (see post linked below for more details).

Note: beta blocks (in this case _AI Chat_ and _Blogroll_ only required to update `block.json`. Other blocks don't use that file for registration yet and required to update the description in `index.js` as well.

### Other information:


## Jetpack product discussion


p1HpG7-oV9-p2

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch.
- Visit `Posts > Add New`.
- Check in the block inserter that the description of at least one of the updated blocks has changed.
- Verify in the code that others changes are correct (no typo, missing punctuation, etc.).
